src/ontology/imports/foodon_product_import.owl

```diff
@@ -34193,6 +34193,105 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001859"/>
+        <obo:IAO_0000115 xml:lang="en">A powder that is prepared by grinding flax seeds.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-05-16T10:02:32</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">flax powder</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">flaxseed powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004277 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004277">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03315708"/>
+        <obo:IAO_0000115 xml:lang="en">A traditional South Indian condiment made with roasted lentils, whole spices and fresh herbs. It is used to flavor and thicken the South Indian soup-like dish known as Rasam.</obo:IAO_0000115>
+        <obo:IAO_0000119 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://www.indianhealthyrecipes.com/rasam-powder-recipe/</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-05-16T10:10:27</dc:date>
+        <rdfs:label xml:lang="en">rasam powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004280 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004280">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003083"/>
+        <obo:IAO_0000115 xml:lang="en">Meat from a calf that is raised on grain, hay, or other solid food, in addition to milk.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-05-16T11:07:43Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">nonformula-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">red veal</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">grain-fed veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004281 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004281">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03301484"/>
+        <obo:IAO_0000115 xml:lang="en">Milk powder that is obtained by removing all of the water from pasteurized skim milk.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-05-16T11:24:35Z</dc:date>
+        <rdfs:label xml:lang="en">skim milk powder</rdfs:label>
```

Comment:
I think this term is a duplicate of http://purl.obolibrary.org/obo/FOODON_03310016 "cow skim milk (powdered)" . That term could use a definition, and the synonym.  It could be made a subclass of FOODON_03301484 too.  Perhaps this term should have "cow skim milk powder" to discriminate from goat or sheep skim milk powder.

Reply:
Oh yes, you're absolutely right. It wasn't underneath 'skim milk food product' so I couldn't find it; I have now added that as a parent class along with synonyms and a definition. Thanks so much.